src/plugin/svipc/sysvipcwrappers.cpp

```diff
@@ -38,6 +38,20 @@
 using namespace dmtcp;
 
 static struct timespec ts_100ms = {0, 100 * 1000 * 1000};
+
+/*
+ * In OpenMPI 2.0, shmdt() is intercepted by modifying libraries' global offset
+ * table, meaning that _real_shmdt() will be redirected into OpenMPI's hook
+ * function, instead of libc's shmdt(). The hook function finally calls syscall()
+ * with the corresponding syscall number. The inside_shmdt variable indicates
+ * if the code is inside our shmdt() wrapper. If so, our syscall wrapper simply
+ * calls _real_syscall(), instead of calling the shmdt() wrapper, resulting in
```

Comment:
Could you please rephrase this sentence? It reads as if calling `_real_syscall()` results in an infinite recursion.